ksqldb-common/src/main/java/io/confluent/ksql/function/types/KsqlLambdaType.java

```diff
@@ -0,0 +1,70 @@
+/*
+ * Copyright 2021 Confluent Inc.
+ *
+ * Licensed under the Confluent Community License (the "License"; you may not use
+ * this file except in compliance with the License. You may obtain a copy of the
+ * License at
+ *
+ * http://www.confluent.io/confluent-community-license
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OF ANY KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+
+package io.confluent.ksql.function.types;
+
+import java.util.Objects;
+
+public final class KsqlLambdaType extends ObjectType {
```

Comment:
nit: do we need to prefix everything with `Ksql`? Personally, I prefer just `LambdaType` 😂

Reply:
Sorry, that was my choice in the POC